### DISCUSSION TITLE
Lastenlösen-Buchungsseite mit schwerwiegenden Fehlern

### DISCUSSION THREAD

**Discussion by RainerBielefeld** [wielebenwir/commonsbooking#1272] [Questions & Answers]
Hallo,

auf der [Lastenlöwen-Webseite](https://www.heinrich-der-lastenloewe.de/)  gibt es ein paar Probleme, zumindest das erste könnte auch irgendwie commonsbooking betreffen:  
  
## 1. Dauerschließung Standort SFU  
![grafik](https://github.com/wielebenwir/commonsbooking/assets/11803749/91f54766-0906-4d18-9038-3c6c51cb8b35)

Standort-Dauerschließung? Falscher Kalendereintrag oder Verleih tatsächlich derzeit geschlossen?
Ich telefonierte mit einem etwas stieseligem Mitarbeiter dort, von dem ich immerhin erfuhr, dass der LL dort steht und gemietet werden kann. Einfach nur Bedienungsfehler des Standortpersonals (ich habe keine Ahnung, wer beim Buchungssysten Standort-Öffnungszeiten einträgt ...)? Oder doch irgendein "Fehler im System"? Ich kann's nicht beurteilen, Lastenlöwenteam hat sich auf meine Reklamation von neulich noch nicht gemeldet. 
  
## 2. Zerrissener Standort "Guten Morgen Buchladen" 
Steht 2x drin, mal mit, mal ohne Lastenrad am Standort.
Verbesserungsmöglichkeit beim Buchungssystem denkbar, die helfen, solche Pannen zu vermeiden?
  
## 3. "BZV Medienhaus"
[Aufforderung](https://www.heinrich-der-lastenloewe.de/cb_location/konzertkasse/) zur Buchungsbestätigung und Abholuhrzeit-Hinweise, obwohl's da anscheinend gar kein Rad gibt ... 
Verbesserungsmöglichkeit beim Buchungssystem denkbar, die helfen, solche Pannen zu vermeiden?

Vielleicht kann das [commonsbooking](https://github.com/wielebenwir/commonsbooking)-Team da etwas helfen?

CU

Rainer

**Comment by RainerBielefeld** [comment]
Hauptproblem "Dauerschließung" ist schon mal gelöst.

**Comment by hansmorb** [comment]
Hallo Rainer, danke für deine Nachricht. Ich glaube aber das gehört hier nicht hin, das sieht mir eher nach einer falschen Konfiguration durch die Initiative aus. Wenn dann müssten die uns schon direkt kontaktieren. Am besten direkt über mail@commonsbooking.org

LG

**Reply by RainerBielefeld** [reply]
Sehe ich auch so!
